nbsdecoder.d.ts

```diff
@@ -28,6 +28,23 @@ export interface NbsPacket {
   payload?: Buffer;
 }
 
+/**
+ * An NBS packet to write to an NBS file
+ */
+export interface WritePacket {
```

Comment:
Let's have the `Nbs` prefix on this too. That way it's easier to tell where it's from when we use it in client code.

```suggestion
export interface NbsWritePacket {
```